.github/workflows/update-package-lock.yaml

```diff
@@ -0,0 +1,27 @@
+name: Update package-lock.json

```

Comment:
@weswigham @andrewbranch Is there a better way to test this than confirming it ran the night after this PR merged?

Reply:
Probably, but I don’t think any of them are as good as the way you mentioned. You can add a `workflow_dispatch` trigger in addition to the `schedule` trigger to be able to run it manually.